packages/react/src/patterns/sections/CTASection/CTASection.js

```diff
@@ -15,34 +16,49 @@ const { prefix } = settings;
  * @param {string} props.heading heading string
  * @param {string} props.copy pattern copy
  * @param {object} props.cta CTA component
+ * @param {string} props.theme theme name
  * @ param {Array} props.items CTA items to the ContentItem
  * @returns {object} JSX Object
  */
-const CTASection = ({ heading, copy, cta, items }) => (
-  <section
-    data-autoid={`${stablePrefix}--cta-section`}
-    className={`${prefix}--cta-section`}>
-    <ContentBlock heading={heading} copy={copy} cta={cta} />
-    <hr className={`${prefix}--horizontal-line`} />
-    <div className={`${prefix}--helper-wrapper`}>
-      <div className={`${prefix}--content-item-wrapper`}>
-        {items.map((item, index) => (
-          <ContentItem
-            key={index}
-            heading={item.heading}
-            copy={item.copy}
-            cta={item.cta}
-          />
-        ))}
+const CTASection = ({ heading, copy, cta, items, theme }) => {
+  /**
+   * sets the class name based on theme type
+   *
+   * @private
+   * @param {string} theme theme type
+   * @returns {string} theme css class names
+   */
+  const _setTheme = theme => {
+    return theme && `${prefix}--cta-section--${theme}`;
+  };
+
+  return (
+    <section
+      data-autoid={`${stablePrefix}--cta-section`}
+      className={classNames(`${prefix}--cta-section`, _setTheme(theme))}>
+      <ContentBlock heading={heading} copy={copy} cta={cta} />
+      <hr className={`${prefix}--horizontal-line`} />
+      <div className={`${prefix}--helper-wrapper`}>
+        <div className={`${prefix}--content-item-wrapper`}>
+          {items.map((item, index) => (
+            <ContentItem
+              key={index}
+              heading={item.heading}
+              copy={item.copy}
+              cta={item.cta}
+            />
+          ))}
+        </div>
       </div>
-    </div>
-  </section>
-);
+    </section>
+  );
+};
 
 CTASection.propTypes = {
   heading: PropTypes.string.isRequired,
   copy: PropTypes.string.isRequired,
   cta: PropTypes.object,
+  theme: PropTypes.string,
```

Comment:
@guilhermelMoraes can you add this to the README?

Reply:
yeah, for sure